e2e-tests/tests/init-deploy/03-read-from-primary.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 30
+commands:
+- script: |
+    set -o errexit
+    set -o xtrace
+
+    source ../../functions
+
+    data=$(run_mysql "SELECT * FROM myDB.myTable" "-h $(get_mysql_primary_service $(get_cluster_name)) -uroot -proot_password")
+
+    kubectl create configmap -n "${NAMESPACE}" 03-read-from-primary --from-literal=data="${data}"
```

Comment:
the comparison should be done during assertion step in `03-assert.yaml`. configmap is used to transfer the data between setup and assert substeps

Reply:
@delgod comparison with configmaps clearly shows the missing data in assertion. otherwise we'll just see the command failure. I think this approach has only upsides and no downsides. What do you think?